server/model.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../database/index');
+
+const bookings = {
+  get: callback => {
+    let query = 'SELECT tables.*, restaurants.name FROM tables, restaurants WHERE restaurants.id = tables.restaurantId;';
+    db.sequelize.query(query, { type: db.sequelize.QueryTypes.SELECT}).then(tables => {
+      callback(null, tables);
+    }).catch(err => {
+      callback(err, null);
+    });
```

Comment:
might be cleaner to write it like this. (just wanted to say it as required to say one improvement. other than that everything looks great)

.query (
)
.then ( 
)
.catch (
)